internal/servers/controller/handlers/authmethods/oidc.go

```diff
@@ -143,14 +138,16 @@ func (s Service) updateOidcInRepo(ctx context.Context, scopeId string, req *pbs.
 func (s Service) authenticateOidc(ctx context.Context, req *pbs.AuthenticateRequest, authResults *auth.VerifyResults) (*pbs.AuthenticateResponse, error) {
 	const op = "authmethod_service.(Service).authenticateOidc"
 	if req == nil {
-		return nil, errors.New(errors.InvalidParameter, op, "nil request")
```

Comment:
@jimlambrt I think I am confused about when errors are "human friendly" vs. not...I think I had an idea that these new error types/schemas were all human friendly but that isn't necessarily the case. In this function though we pass back what comes back from the function directly to the response handler so I think here it's correct to use human-readable ones.